sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/FileSystemAPITest.groovy

```diff
@@ -242,6 +242,23 @@ class FileSystemAPITest extends APISpec {
         thrown(DataLakeStorageException)
     }
 
+    def "Exists"() {
+        when:
+        fsc = primaryDataLakeServiceClient.getFileSystemClient(generateFileSystemName())
+        fsc.create()
+
+        then:
+        fsc.exists()
+    }
+
+    def "Does not exist"() {
```

Comment:
All tests generally start with the method they are testing for easier searching followed by something that describes the scenario/behavior being tested, so I'd name it something like "Exists not exist"